platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -619,7 +619,8 @@ MGL_EXPORT
 
 /**
  Value to use for a text label. Feature properties are specified using tokens
- like {field_name}.
+ like {field_name}.  (Token replacement is only supported for literal text-field
+ values--not for data-driven property functions.)
```

Comment:
It’s a good point to document. There’s no harm in using one term in the style specification and overriding it here to refer to the specific class; that’s what style-spec-overrides-v8.json was designed for.